src/Storefront/Controller/FrontendProxyController.php

```diff
@@ -77,19 +77,6 @@ public function __construct(
         $this->donateController = $donateController;
     }
 
-    /**
-     * @Route(
-     *     "/adyen/proxy-switch-context",
-     *     name="payment.adyen.proxy-switch-context",
-     *     defaults={"XmlHttpRequest"=true, "csrf_protected": false},
-     *     methods={"PATCH"}
-     * )
-     */
-    public function switchContext(RequestDataBag $data, SalesChannelContext $context): ContextTokenResponse
```

Comment:
This is a public endpoint. Even though we are not using it in the headful version, it may be used in headless. What would you think adding `deprecated` tag instead of removing it?